src/ui/public/directives/rows.js

```diff
@@ -40,10 +40,10 @@ module.directive('kbnRows', function ($compile, $rootScope, getAppState, Private
         let $cellContent;
 
         if (contents instanceof AggConfigResult) {
+          const field = contents.aggConfig.getField();
           const isCellContentFilterable =
-            contents.type === 'bucket'
-            && contents.aggConfig.getField()
-            && contents.aggConfig.getField().filterable;
+            _.get(contents, 'aggConfig.type.createFilter')
+            && (!field || field.filterable);
 
```

Comment:
@lukasolson lemme know if you wanna take a crack at this, if so I'll hold off on giving it another look. Otherwise I'll do another review first thing tomorrow.